style.css

```diff
@@ -121,8 +112,237 @@ nav {
   margin: 0 12px;
 }
 
-/* Scroll button */
 .scroll-button {
   margin-top: 83px;
   align-items: center;
+  margin-bottom: 63px;
+}
+
+/* Work section */
+.work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  height: 100vh;
+  margin: 10px 0 0 5px;
+}
+
+/* Project title work */
+.title h2 {
+  color: var(--headline-color-1);
+  height: 97px;
+  font-weight: 700;
+  line-height: 52px;
+  font-size: 40px;
+  margin-top: 63px;
+  margin-right: 28px;
+  margin-left: 26px;
+  margin-bottom: 24px;
+}
+
+.indicator {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 62px;
+}
+
+/* Grid container work */
+.project-grid {
+  display: grid;
+  grid-template-rows: 252px auto 48px;
+  border-radius: 8px;
+  border: 1px solid var(--project-background-color);
+  margin-bottom: 29px;
+}
+
+.project-image {
+  background: var(--project-background-color);
+}
+
+.project-title {
+  color: var(--header-mobile-color);
+  margin-top: 16px;
+  margin-bottom: 16px;
+}
+
+.project-skills {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.project-skills a {
+  color: var(--header-mobile-color);
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  padding: 11.6px;
+  border-radius: 4px;
+  background: var(--skills-background-color);
+}
+
+.project-skills ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  width: 122px;
+  height: 48px;
+  margin-left: calc(50% - 122px / 2 + 0.5px);
+  margin-top: 29px;
+  margin-bottom: 24px;
+  background-color: var(--project-button-color);
+  border-radius: 4px;
+  border: none;
+}
```

Comment:
- [ ] [OPTIONAL] Great work :muscle: How about adding some nice interactive elements to your buttons by implementing the `:hover` and `:active` states on your button just you have a guide on the wireframe? I think it will improve your design and User experience a lot.
_Here is an example of what the color palette looks like on the wireframe:_
![image](https://user-images.githubusercontent.com/99715640/235295333-b5feffdd-2d9c-404d-9d82-beda8acc9ebd.png)

Reply:
Alright @AbayomiOlaoye. Thank for reminding me that! Let me do that right away